donkeycar/parts/oled.py

```diff
@@ -1,43 +1,51 @@
-import Adafruit_SSD1306
-
-from PIL import Image
-from PIL import ImageDraw
-from PIL import ImageFont
 import subprocess
 import time
+from board import SCL, SDA
+import busio
+from PIL import Image, ImageDraw, ImageFont
+import adafruit_ssd1306
 
 class OLEDDisplay(object):
     '''
     Manages drawing of text on the OLED display.
     '''
-    def __init__(self, bus_number=1):
```

Comment:
We need to provide the `I2C_BUSNO` on a Jetson Nano for this to work.

Reply:
@tikurahul The way the Adafruit CircuitPython library framework works is that all those details, such as bus number, are automatically defined as part of the overall framework at installation time so they don't have to be redefined for each part. So there is no need (or way) to define a bus number in the new OLED library. 

That said, I have now added the installation of the CircuitPython framework to the overall Nano setup so people won't have to do it manually if they're using an OLED

Reply:
That is awesome ! Thanks for clearing that up.